src/azure-cli/azure/cli/command_modules/batch/tests/latest/test_batch_data_plane_commands.py

```diff
@@ -320,7 +320,7 @@ def test_batch_pools_and_nodes(
 
         # test create pool from JSON file
         self.kwargs['json'] = self._get_test_data_file('batch-pool-create.json').replace('\\', '\\\\')
-        self.batch_cmd('batch pool create --json-file {json}')
+        self.batch_cmd('batch pool create --json-file "{json}"')
```

Comment:
@bgklein can you review changes in `test_batch_data_plane_commands.py` and approve if it looks good to you?